eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```diff
@@ -75,3 +75,30 @@ services:
             - '@ezpublish.spi.search'
         tags:
             - { name: kernel.event_subscriber }
+
+    eZ\Publish\Core\FieldType\Image\ImageThumbnailStrategy:
+        arguments:
+            $variationHandler: '@ezpublish.image_alias.imagine.variation.imagine_alias_generator'
+            $variationName: 'medium'
+
+    eZ\Publish\Core\FieldType\ImageAsset\ImageAssetThumbnailStrategy:
+        lazy: true
+        arguments:
+            $thumbnailStrategy: '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\ThumbnailChainStrategy'
+            $contentService: '@ezpublish.api.service.content'
+
+    eZ\Publish\Core\Repository\Strategy\ContentThumbnail\Field\ContentFieldStrategy:
+        arguments:
+            $strategies:
+                ezimage: '@eZ\Publish\Core\FieldType\Image\ImageThumbnailStrategy'
+                ezimageasset: '@eZ\Publish\Core\FieldType\ImageAsset\ImageAssetThumbnailStrategy'
+
+    eZ\Publish\Core\Repository\Strategy\ContentThumbnail\FirstMatchingFieldStrategy:
+        arguments:
+            $fieldTypeService: '@ezpublish.api.service.field_type'
+            $contentFieldStrategy: '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\Field\ContentFieldStrategy'
+
+    eZ\Publish\Core\Repository\Strategy\ContentThumbnail\ThumbnailChainStrategy:
+        arguments:
+            $strategies:
+                0: '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\FirstMatchingFieldStrategy'
```

Comment:
It looks very weird, please use service tagging, e.g. `ezplatform.spi.content.thumbnail` (made more sense when there was another one, with 255 key, but still suggests it's a proper direction`).